dependencies.props

```diff
@@ -2,13 +2,15 @@
   <!-- Source of truth for dependency tooling: the commit hash of the dotnet/versions master branch as of the last auto-upgrade. -->
   <PropertyGroup>
     <CoreFxCurrentRef>534770cbd98c4083de4d93b809f2e0968f0f856a</CoreFxCurrentRef>
-    <WCFCurrentRef>c9a03fe0c3309508cb509a4ffdd6dc8b79ed525e</WCFCurrentRef>
+    <WCFCurrentRef>a2ba4d6141ef25bdb4c2d2c014b9e1f305a0ea90</WCFCurrentRef>
     <StandardCurrentRef>8f8313cbe4b0f4d89c4d0835e08e3e340a18eec0</StandardCurrentRef>
+    <CoreSetupCurrentRef>c2c072242cb0bdea99efb7349d69492f83200a7d</CoreSetupCurrentRef>
   </PropertyGroup>
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
     <CoreFxExpectedPrerelease>preview2-25405-01</CoreFxExpectedPrerelease>
+    <WcfExpectedPrerelease>preview2-25413-01</WcfExpectedPrerelease>
```

Comment:
Technically you can leave out this property (and its upgrader), and project.json upgrades would still happen. Up to you if you think this is useful info to have in dependencies.props! E.g. CoreFX has some \*ExpectedPrerelease properties, but Core-Setup doesn't have any.